test/verify/check-storage-lvm2

```diff
@@ -230,6 +230,7 @@ class TestStorage(StorageCase):
         # Put a partition table on disk1 and disk2
         m.execute('parted -s %s mktable gpt' % disk1)
         m.execute('parted -s %s mktable gpt' % disk2)
+        m.execute("udevadm settle")
```

Comment:
The dialog retry loop below should be able to take care of this, no? Maybe we need to make the `expect` condition more specific.

Reply:
It seems it actually uses the raw device, and overwrites the partition table. But indeed we could try to make the match string more specific to ensure that it sees the partitioning, I'll have a look.

Reply:
check-storage-used etc have the udevadm settle so that the mkfs can succeed, I think, and it might indeed be enough here, too. But we do have the retry loop already, and that would be even more robust...